deposit/hal/forms.py

```diff
@@ -21,21 +21,55 @@
 from __future__ import unicode_literals
```

Comment:
Imports can be sorted.

*PyImportSortBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/deposit/hal/forms.py
+++ b/deposit/hal/forms.py
@@ -20,12 +20,11 @@
 
 from __future__ import unicode_literals
 
+from autocomplete.widgets import Select2
 from crispy_forms.helper import FormHelper
 from deposit.forms import BaseMetadataForm
 from django import forms
 from django.utils.translation import ugettext as __
-
-from autocomplete.widgets import Select2
 
 HAL_TOPIC_CHOICES = [
     ('CHIM', __('Chemistry')),
```